website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,128 @@
+Karpenter carries out its duties by:
+
+* Watching for pods that the Kubernetes scheduler has marked as unschedulable
+* Evaluating the requirements (capacity, features, and taints) identified by the unscheduable pods
+* Provisioning nodes that meet the requirements of the unschedulable pods
+* Scheduling the unschedulable pods to run on the new nodes
+* Removing the nodes when the nodes are no longer needed
+
+In many cases, you can configure a simple Karpenter provisioner when it is first installed and not change it again. 
+Other times, you might want to continue to tweak the provisioner or even create multiple provisioners for a cluster where you tailor the requirements of different teams to use specific instance types, zones, or computer architectures.
+Likewise, you might want to instruct the provisioner to take advantage of underlying cloud features, such as AWS spot instances.
+
+As someone deploying pod workloads, you can make specific requests for capacity and features you want from the nodes running your pods.
+If the Kubernetes scheduler is unable to meet those requirements, Karpenter is designed to quickly create the best possible nodes to meet those needs.
+
+## Karpenter concepts
+
+The following are some key concepts you should understand about Karpenter.
+
+### Karpenter provisioners
+
+A Karpenter provisioner controller runs on a Kubernetes cluster.
+Its job is to add nodes to handle unschedulable pods, schedule pods on those nodes, and remove the nodes when they are not needed.
+The controller's work is configured through [Provisioner CRDs](/docs/provisioner-crd/).
+
+Rather than take over the process of provisioning nodes and scheduling pods, the provisioner controller integrates with the Kubernetes scheduler, cluster autoscaler, and related Kubernetes features to make sure that pods are properly and efficiently deployed.
+It also works through Kubernetes to request needed assets from the cloud provider.
+
+Here are some attributes of the Karpenter provisioner:
+
+* **Unschedulable pods**: Karpenter only looks at pods that the Kubernetes scheduler identifies as unschedulable.
+If the Kubernetes scheduler is able to schedule a pod, the Karpenter provisioner takes no action on it.
+* **Provisioner CRD**: Provisioner CRDs define how the provisioner controller behaves.
+Each CRD contains constraints that impact the nodes that can be provisioned and attributes of those nodes (such timers for removing nodes).
+See [Provisioner CRD](/docs/provisioner-crd/) for a description of settings.
+* **Well-known labels**: The CRD can use well-known Kubernetes labels to allow pods to request only certain instance types, computer architectures, operating systems, or other attributes when creating nodes.
+* **Time to live**: A provisioner can also include time-to-live values to indicate when nodes should be decommissioned after a set amount of time from when it was created or after it becomes empty of deployed pods.
```

Comment:
This one feels a bit in the weeds to me. We currently only have two controls, but this will balloon over time beyond TTLs. Worth mentioning limits (not yet implemented) if anything at all.

Reply:
I've added a deprovisioning task section to the new outline. I'll just give a brief mention here and point to the new task.